dm/config/subtask.go

```diff
@@ -204,14 +204,16 @@ type SubTaskConfig struct {
 	//  treat it as hidden configuration
 	IgnoreCheckingItems []string `toml:"ignore-checking-items" json:"ignore-checking-items"`
 	// it represents a MySQL/MariaDB instance or a replica group
-	SourceID                string `toml:"source-id" json:"source-id"`
-	ServerID                uint32 `toml:"server-id" json:"server-id"`
-	Flavor                  string `toml:"flavor" json:"flavor"`
-	MetaSchema              string `toml:"meta-schema" json:"meta-schema"`
-	HeartbeatUpdateInterval int    `toml:"heartbeat-update-interval" json:"heartbeat-update-interval"`
-	HeartbeatReportInterval int    `toml:"heartbeat-report-interval" json:"heartbeat-report-interval"`
-	EnableHeartbeat         bool   `toml:"enable-heartbeat" json:"enable-heartbeat"`
-	Meta                    *Meta  `toml:"meta" json:"meta"`
+	SourceID   string `toml:"source-id" json:"source-id"`
+	ServerID   uint32 `toml:"server-id" json:"server-id"`
+	Flavor     string `toml:"flavor" json:"flavor"`
+	MetaSchema string `toml:"meta-schema" json:"meta-schema"`
+	// deprecated start
+	HeartbeatUpdateInterval int  `toml:"heartbeat-update-interval" json:"heartbeat-update-interval"`
+	HeartbeatReportInterval int  `toml:"heartbeat-report-interval" json:"heartbeat-report-interval"`
+	EnableHeartbeat         bool `toml:"enable-heartbeat" json:"enable-heartbeat"`
+	// deprecated end
```

Comment:
my clever GoLand can recognize `// deprecated` one line before the field, it can't recognize `// deprecated start` and `// deprecated end`